lib/country_city_validators/us_city_validator.rb

```diff
@@ -62,9 +62,13 @@ module CountryCityValidators
     U.S.\ Virgin\ Islands
   ).to_set
 
+  US_DISTRICTS = %w(
+    District\ Of\ Columbia
```

Comment:
"D.C." feels a lot more natural to me than spelling out "District of Columbia", consider switching to that?